[ISSUE-1138] Import LogBack JSON layout into tmail (#1250)
linagora/tmail-backend: logback-contrib/README.md

# logback-contrib  [![GitHub release](https://img.shields.io/github/release/qos-ch/logback-contrib.svg?maxAge=2592000)]() [![Build Status](https://travis-ci.org/qos-ch/logback-contrib.svg?branch=master)](https://travis-ci.org/qos-ch/logback-contrib)

> https://github.com/qos-ch/logback-contrib/wiki
> 
> #### Build Instructions
> Run the following command to build:
> 
> ```
> mvn clean install
> ```

Reviewer: ```suggestion
```

#### License
```

> This program and the accompanying materials are licensed under

Reviewer: ```suggestion
This maven module subtree and the accompanying materials are licensed under
```

> #### License
> ```
> This program and the accompanying materials are licensed under
> the terms of the Eclipse Public License v1.0 as published by
> the Eclipse Foundation.
> ```

Reviewer: ```suggestion
#### License
```
This maven module subtree and the accompanying materials are licensed under
the terms of the Eclipse Public License v1.0 as published by
the Eclipse Foundation.
```
```

# logback-contrib  [![GitHub release](https://img.shields.io/github/release/qos-ch/logback-contrib.svg?maxAge=2592000)]() [![Build Status](https://travis-ci.org/qos-ch/logback-contrib.svg?branch=master)](https://travis-ci.org/qos-ch/logback-contrib)

https://github.com/qos-ch/logback-contrib/wiki

#### Build Instructions
Run the following command to build:

```
mvn clean install
```

#### License
```
This program and the accompanying materials are dual-licensed under
either the terms of the Eclipse Public License v1.0 as published by
the Eclipse Foundation

  or (per the licensee's choosing)

under the terms of the GNU Lesser General Public License version 2.1
as published by the Free Software Foundation.

> ```

Reviewer: Idem for the license clause in here
